src/nexusrpc/_types.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import annotations
+
+from typing import TypeVar
+
+# Operation input type
```

Comment:
Can you make sure these comments are considered docstrings?

Reply:
Done (I didn't know that VSCode/LSP took ad-hoc strings as docstrings in so many Python contexts)